.travis.yml

```diff
@@ -1,52 +1,31 @@
 language: java
-jdk:
-- oraclejdk8
 
 dist: trusty
 sudo: required
 
 install: true
 
-
-jobs:
+matrix:
   include:
-  - stage: build
-    script: bash scripts/build.sh
-  - stage: sample-bank-service
-    script:
-    - mvn -f simulator-samples/sample-bank-service/pom.xml verify
-  - stage: sample-combined
-    script:
-    - mvn -f simulator-samples/sample-combined/pom.xml verify
-  - stage: sample-jms
-    script:
-    - mvn -f simulator-samples/sample-jms/pom.xml verify
-  - stage: sample-jms-fax
-    script:
-    - mvn -f simulator-samples/sample-jms-fax/pom.xml verify
-  - stage: sample-mail
-    script:
-    - mvn -f simulator-samples/sample-mail/pom.xml verify
-  - stage: sample-rest
-    script:
-    - mvn -f simulator-samples/sample-rest/pom.xml verify
-  - stage: sample-swagger
-    script:
-    - mvn -f simulator-samples/sample-swagger/pom.xml verify
-  - stage: sample-ws
-    script:
-    - mvn -f simulator-samples/sample-ws/pom.xml verify
-  - stage: sample-ws-client
-    script:
-    - mvn -f simulator-samples/sample-ws-client/pom.xml verify
-  - stage: sample-wsdl
-    script:
-    - mvn -f simulator-samples/sample-wsdl/pom.xml verify
+    # OracleJDK - Java 8 (main build with Sonar)
+    - env: SONAR='true'
+      jdk: oraclejdk8
+    # OpenJDK - Java 11 (lts)
+    - env: SONAR='false' BUILD_OPTS='-Djava.version=11'
+      jdk: openjdk11
+    # OpenJDK - Java 12 (latest)
+    - env: SONAR='false' BUILD_OPTS='-Djava.version=12'
+      jdk: openjdk12
```

Comment:
indentation is not equal..
```
matrix:
  include:
  - env: ...
```
vs
```
script:
  - if
```

Reply:
The lists are now indented, which is more consistent